config/deploy/dev.rb

```diff
@@ -0,0 +1,6 @@
+require_relative 'server_settings'
+
+server 'sdr-services-dev.stanford.edu', user: 'sdr2service', roles: %w{app}
+set :deploy_to, "/var/sdr2service/#{ENV['SDR_APP']}"
+
+Capistrano::OneTimeKey.generate_one_time_key!
```

Comment:
This is all moving in the right direction, but I have vague recollections that some ENV values might be used in the app code.  I just haven't set aside time to work on this, as I wasn't included in this work, but I'm the only dev mostly familiar with this repo/app (yes, we need to change that, but don't ask me when that's going to happen; I've asked many times).